tests/functional/integration-tests.js

```diff
@@ -0,0 +1,91 @@
+/*globals casper */
+
+// Test the various integration pages.
+//
+// We limit ourselves to one test per integration page. This may seem low,
+// but keep in mind that functional tests are not supposed to be as exhaustive
+// as unit tests. Instead, their role is to look at the system as a whole.
+// Nevertheless, we test a variety of features of FormatJS and multiple locales
+// to try and get the highest possible coverage and confidence that the site is
+// working properly.
+//
+// All the tests basically do the same thing. Load a page, modify the value
+// of the locale selection combo box, and check the output of the corresponding
+// example with a static value. Because these tests are all the same, they are
+// parameterized below.
+
+'use strict';
+
+var testData = {
```

Comment:
I don't know, I could not find a similar example. I would not want to put this in the `Gruntfile.js`. But I could extract it into a separate file, if needed. Originally, I had 4 (almost identical) test files, which gave me the idea to write a single test and parameterize it.